index.js

```diff
@@ -46,13 +54,27 @@ function Streampub (opts) {
 }
 inherits(Streampub, Transform)
 
+// Theoretically this could be done in _flush, but somehow this generates an error
+// this is a workaround for that error ('no write after end')
+Streampub.prototype._end = Streampub.prototype.end;
```

Comment:
Yeah I tried that, but somehow it did not work correctly. If I remember correctly it did seem to work but the `cover.html` did not end up in the zip file.

I should look at it again. It didn't feel right doing it like this.